dashboard/test/ui/features/projectsLegacy.feature

```diff
@@ -0,0 +1,51 @@
+Feature: Projects
+
+# TODO - do we need to make sure we have a valid channel
+# TODO - IE9
+
+@stop_after_failure
```

Comment:
These tests take an unfortunately long time to run. Each example can wait up to 30 seconds to see the redirect. This becomes particularly painful in cases where something goes wrong. @stop_after_failure makes it so that as soon as we get one failure, we stop running other examples.